XenAdmin/TabPages/ManageUpdatesPage.Designer.cs

```diff
@@ -423,5 +467,10 @@ private void InitializeComponent()
         private System.Windows.Forms.DataGridViewTextBoxColumn ColumnDate;
         private System.Windows.Forms.DataGridViewTextBoxColumn ColumnWebPage;
         private System.Windows.Forms.TableLayoutPanel tableLayouPanel5;
+        private System.Windows.Forms.ToolStripSeparator toolStripSeparator;
+        private System.Windows.Forms.ToolStripSplitButton toolStripSplitButtonView;

```

Comment:
The control for this use should be ToolStripDropDownButton, not ToolStripSplitButton. With the ToolStripSplitButton the user selects an item and this item becomes the default, so next time they want to repeat the action they can click it directly without having to open the dropdown again. However, here you only select one of the two items, you don't need to repeat an action, therefore you need the ToolStripDropDownButton.

Also, when you select one of the dropdown items, this item should stay highlighted. Currently, the first item is always highlighted, no matter which one you click.

(We have a similar button on the Snapshots TabPage, you can have a look there to see how it works).